---
layout: post
title: Spring papers news
wordpress_id: 2067
wordpress_url: http://lab.stajich.org/home/?p=2067
categories: []
tags: []
---
<!-- wp:image {"id":2073} --><figure class="wp-block-image">![](/images/wp_upload/2019/04/IMG_0653_smaller.jpg)<figcaption>We enjoyed a wonderful spring flower season here in Southern California <br />(Harford Springs Riverside County Regional Park)</figcaption></figure><!-- /wp:image --><!-- wp:paragraph -->
Several publications were accepted and published in last few months and want to highlight the hard work from lab members and collaborators.
<!-- /wp:paragraph --><!-- wp:paragraph -->
[Jinfeng](http://lab.stajich.org/home/people/jinfeng-chen/) lead article published on the [localized burst of Rice Transposable Elements ](https://www.nature.com/articles/s41467-019-08451-3)_[Ping](https://www.nature.com/articles/s41467-019-08451-3)_[ and ](https://www.nature.com/articles/s41467-019-08451-3)_[Pong](https://www.nature.com/articles/s41467-019-08451-3)_[ in only a subset of the domesticated lineages](https://www.nature.com/articles/s41467-019-08451-3) and clues to what changed to drive their increased activity. 
<!-- /wp:paragraph --><!-- wp:image {"align":"center"} --><div class="wp-block-image"><figure class="aligncenter">![](https://media.springernature.com/lw900/springer-static/image/art%3A10.1038%2Fs41467-019-08451-3/MediaObjects/41467_2019_8451_Fig2_HTML.png)<figcaption>Variation of _mPing, Ping_, and&nbsp;_Pong_ element copy number in rice subgroups [https://www.nature.com/articles/s41467-019-08451-3/figures/2](https://www.nature.com/articles/s41467-019-08451-3/figures/2)</figcaption></figure></div><!-- /wp:image --><!-- wp:paragraph -->
A fun collaboration lead by [Adam Bewick](https://twitter.com/bewickaj) and [Bob Schmitz](http://schmitzlab.genetics.uga.edu/) we called #MycoMeth explored [diversity of Cytosine DNA Methylation in Fungi](https://www.nature.com/articles/s41559-019-0810-9) looking at the 5-methylcytosine (5mC) marks and evolution of the fungal 5mC methyltransferases .
<!-- /wp:paragraph --><!-- wp:image {"id":2069} --><figure class="wp-block-image">![](/images/wp_upload/2019/04/41559_2019_810_Fig2_HTML.png)<figcaption>Genomewide 5mC profiles [https://www.nature.com/articles/s41559-019-0810-9/figures/2](https://www.nature.com/articles/s41559-019-0810-9/figures/2)</figcaption></figure><!-- /wp:image --><!-- wp:paragraph -->
Two Zygolife project papers. One on [genome evolution and phylogenic position of Endogone - a truffle forming Mucoromycotina fungus. ](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.15613)This was quite challenging but interesting project to tackle, lead by [Ying Chang](https://joeyspataforalab.weebly.com/lab-members.html) in the [Spatafora Lab](https://joeyspataforalab.weebly.com/). Dr Chang took a metagenomics approach to capture these unculturable fungi followed by bioinformatic analyses to isolate the fungal and Mucoromycotina component of the dataset. 
<!-- /wp:paragraph --><!-- wp:image {"id":2070} --><figure class="wp-block-image">![](/images/wp_upload/2019/04/nph15613-fig-0003-m.jpg)<figcaption>Phylogenetic placement and divergence time estimation of Endogonales_,_&nbsp;and the evolution of fungal plant‐cell‐wall‐degrading enzymes (PCWDEs)</figcaption></figure><!-- /wp:image --><!-- wp:paragraph -->
A second paper lead by [William Davis](http://umich.edu/~mycology/people.html) from [Tim James](http://umich.edu/~mycology/) lab on the [genomes and phylogeny of Zoopagomycotina fungi](https://doi.org/10.1016/j.ympev.2019.01.006) which include nematophagous and mycoparasites.
<!-- /wp:paragraph --><!-- wp:paragraph -->

<!-- /wp:paragraph --><!-- wp:image {"id":2071} --><figure class="wp-block-image">![](/images/wp_upload/2019/04/1-s2.0-S105579031830558X-gr5_lrg.jpg)<figcaption>[Fig 5. ](https://www.sciencedirect.com/science/article/pii/S105579031830558X#f0025)Ancestral state reconstructions of trophic state on the best ML tree (A) and the ASTRAL consensus tree (B) using maximum likelihood. Pie graphs at nodes show the proportional likelihood of each character state.</figcaption></figure><!-- /wp:image --><!-- wp:paragraph -->
Jason participated in a Marine Fungi workshop on the unexplored diversity in the marine environments and[ co-authored this perspective on the opportunities and challenges](https://mbio.asm.org/content/10/2/e01189-18) in characterizing and studying these fungi.
<!-- /wp:paragraph --><!-- wp:image --><figure class="wp-block-image">![](https://mbio.asm.org/content/mbio/10/2/e01189-18/F1.large.jpg)<figcaption>[Morphological diversity of fungi](https://mbio.asm.org/content/10/2/e01189-18/figures-only) collected from a biotic host. Fungal collection isolated from a marine sponge,&nbsp;_Ircinia variabilis_&nbsp;(formerly&nbsp;_Psammocinia_ sp.). </figcaption></figure><!-- /wp:image --><!-- wp:paragraph -->
Former postdoc [Ousmane Cissé](https://scholar.google.com/citations?user=sjUY-OYAAAAJ&hl=en) [developed FGMP,](https://github.com/stajichlab/FGMP) a tool for assessing completeness of fungal genomes which uses both conserved protein coding markers that can include multicopy gene families and conserved non-coding DNA regions.  This [was published this month as well](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2782-9), congrats Ousmane!
<!-- /wp:paragraph --><!-- wp:image {"id":2072} --><figure class="wp-block-image">![](/images/wp_upload/2019/04/12859_2019_2782_Fig2_HTML.png)<figcaption>[Fig 2. ](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2782-9#Fig2)Estimation of genome completeness in fungal genomes. A comparison of genome completeness by multiple software tools using initial and latest genome assembly versions of 45 fungi. Genome completeness expressed as a percentage of expected markers (y-axis) is plotted against assembly size in megabases (x-axis).</figcaption></figure><!-- /wp:image -->
